spec/support/shared_examples/localized_request.rb

```diff
@@ -0,0 +1,9 @@
+shared_examples_for 'localized request' do |action|
+  it 'calls I18n.with_locale in requests' do
+    allow(I18n).to receive(:with_locale)
+
+    get action
+
+    expect(I18n).to have_received(:with_locale).with(:en)
+  end
+end
```

Comment:
@brunoocasali would it be very difficult to also test loading a page with another locale? So far our test quite runs strictly against the site in English. I would love to have some test for a locale other than English, even if it was a very basic or simple test.

If you could do that, it would be much appreciated. If it's a lot of work, let me know, and I'll still accept this pull request without it.

Thank you, and best regards.

\- DeeDeeg

Reply:
Hi @DeeDeeG, thanks for your review! 

Don't you think that way we could start testing some code that are already tested by the framework?

The test in this PR just guarantees that every request will contain some kind of language definition. 
When we add a multi-language switcher like a querystring called "lang" for example we could add another spec here, to check.

if the url contains the "lang" querystring all the messages will be translated properly.

Reply:
It should be fairly easy to set the locale within a spec's code.

But I can see how that would be unrelated to this pull request. Perhaps when I re-add the language switcher feature, I will `@mention` you to see if you might be interested in adding some tests. No obligation, but certainly it would be appreciated if you wanted to help with that.

Thanks for this, I'm going to merge now.

Reply:
@DeeDeeG of course! just mention me and I see what can I do ;)